roles/backpack/templates/backpack.yml

```diff
@@ -20,6 +20,12 @@ spec:
       tolerations:
       - key: node-role.kubernetes.io/master
         effect: NoSchedule
+{% if metadata.label is defined %}
+      nodeSelector:
```

Comment:
you are right, NodeSelector list implies an AND, but there is this other way maybe?   [From K8S docs](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/):

"If you specify multiple nodeSelectorTerms associated with nodeAffinity types, then the pod can be scheduled onto a node if one of the nodeSelectorTerms can be satisfied."    That sounds like an OR.   But I don't know how to do it.  If it becomes a big issue I'll do the work to find out how, but for now I think I can make use of this the way it is.

Reply:
Ack. Let me take a look and see if its an  easy implementation